components/Head/index.jsx

```
@@ -66,28 +69,61 @@ export const Head = () => {
-    
-      try {
-        const response = await api.sendMailAdmin("/mail/collaborator", {
-          subject: "Quero ser Head",
-          data: { ...data },
-        });
-
-        if (response.status !== 200) {
-          throw new Error("Não foi possível enviar a requisição");
+      
+      {!dataAccepted && setShowDataModal(true)}
+  
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)